contracts/ligo/src/minter_collection/editions/README.md

```diff
@@ -0,0 +1,48 @@
+# Editions variant of FA2
+
+## Intro
+
+We want to mint many _editions_ of an NFT that share the same metadata,
+but with unique serial numbers, e.g. in a _limited run_.
+With contracts not-specialized to editions, this results in redundant copies of the metadata that's shared between editions in a run.
+We also want the token Ids of tokens in an edition set to be consecutive, to make reasoning about them easier. 
+
+This propsal allows editions creators to easily mint an "unlimited" (limited only by big_map storage limits) number of editions and then distribute them concurrently. 
+
+## Storage
+
+- FA2 + TZIP-16 storage
+
+- Editions-specific storage
+  + `current_token_id : nat`
+  + `current_edition_id : nat`
+  + `editions_metadata`
+
+```ocaml
+editions_metadata :=
+  big_map
+    (nat : edition_id)
+    (token_metadata
+    , address : creator
+    , initial_token_id : nat
+    , nat : number_of_editions
+    , nat : number_of_editions_to_distribute)
+```
+## Entrypoints
+
+- `mint_editions : list mint_edition`
+  + Only admin can call this entrypoint if admin is configured
```

Comment:
if admin's not configured, should we limit edition size, etc? I think it may be best to have either a minter set or fixed admin for this version to keep it simple.

Reply:
I moved to  a `multi_admin` approach where I use the `multi_admin` module version of the standard fa2 nft contract